docs/features/techdocs/getting-started.md

```diff
@@ -230,7 +230,7 @@ You can do so by including the following lines in the last step of your
 
 ```Dockerfile
 RUN apt-get update && apt-get install -y python3 python3-pip
-RUN pip3 install mkdocs-techdocs-core==0.0.16
+RUN pip3 install mkdocs-techdocs-core==0.2.1
```

Comment:
Nice! I wonder if it's also possible to make the section below a bit more visible, do you have ideas?

> Please be aware that the version requirement could change, you need to check our
[`Dockerfile`](https://github.com/backstage/techdocs-container/blob/main/Dockerfile)
and make sure to match with it.

Reply:
I haven't had a chance to dig into this package's contents yet, but this doc-based version pinning is going to be pretty brittle even if we put this in a big, blinking marquee. This is definitely a bigger thing, but my first inclination would be to see whether the APIs have stabled enough to be able to do some more fuzzy pinning. Perhaps something like:
```shell
pip3 install mkdocs-techdocs-core>=0.2.1
```
Or maybe a pin with a greater than + less than range specified. That may reduce the risk of staleness if things have stabilized enough to do so. This also means that versioning has to be disciplined with mkdocs-techdocs-core if it isn't already as well!

Reply:
Hmm...  Yeah, maintaining semver expectations on that plugin is going to be tough.  The API surface area is _really_ fuzzy (markdown syntax on the input side, and HTML markup on the output side).  Until we can find a practical way to define and test the interface, probably better to have occasionally outdated instructions that are more easily reproduced than an underlying dependency that could break your docs on your next deploy.